### PR TITLE
Add unit tests for XSLPipeline

### PR DESCRIPTION
**Pull request by vasiliy-mikhailov**
Additive unit tests for `XSLPipeline` — edge cases and current behavior pinned with explicit assertions. No existing test or production code changed.

Verified green under Java 11 (`mvn -pl . test -Dtest=XSLPipelineTest` → Tests run: 4, Failures: 0, Errors: 0).

### CODE REVIEW

File: src/test/java/com/lyncode/xoai/tests/util/XSLPipelineTest.java [hunk 1]
```diff
@@ -15,6 +16,7 @@
 import static org.junit.Assert.assertThat;
 
 public class XSLPipelineTest extends XmlTest {
+    private static final TransformerFactory tFactory = TransformerFactory.newInstance();
```
Comment (tdonohue) [inline src/test/java/com/lyncode/xoai/tests/util/XSLPipelineTest.java]: This variable and the import statement above are both unused

File: src/test/java/com/lyncode/xoai/tests/util/XSLPipelineTest.java [hunk 2]
```diff
@@ -38,4 +40,13 @@ public void shouldTransformWithoutXmlDeclarationOnTop() throws TransformerExcept
         underTest.apply(identityTemplate());
         assertThat(IOUtils.toString(underTest.getTransformed()), not(containsString("<?xml")));
     }
+
+    @Test
+    public void shouldSupportMethodChainingOnApply() throws TransformerException, IOException {
+        String result = IOUtils.toString(
+                new XSLPipeline(new ByteArrayInputStream(TEST_XML.getBytes()), true)
```
Comment (tdonohue) [inline src/test/java/com/lyncode/xoai/tests/util/XSLPipelineTest.java]: Could be replaced with `new XSLPipeLine(input, true)`, as that would mirror the other tests better.

**Review by tdonohue — CHANGES_REQUESTED** [review]
@vasiliy-mikhailov : Thanks for the contribution. However, this is adding an unnecessary import and unused variable.  Other minor suggestions inline.

**Review by tdonohue — APPROVED** [review]
👍 Thanks @vasiliy-mikhailov .  This small improvement now looks good to me.
